bmu/Src/fanControl.c

```diff
@@ -65,6 +65,7 @@ HAL_StatusTypeDef setFan()
   
   FanPeriod = duty;
   sendCAN_BMU_FanPeriod();
+  vTaskDelay(3); // Added to prevent CAN mailbox full
```

Comment:
We have to be very careful about where we add delays in functions bc you don't want to accidently increase task period. I would suggest adding the delay after the calls to this function instead if you need to do it this way

Reply:
See previous comments for adding delays